packages/app/src/app.ts

```diff
@@ -79,7 +79,8 @@ export class App<Req extends Request = Request, Res extends Response = Response>
       ...options.settings
     }
     this.applyExtensions = options?.applyExtensions
-    this.attach = (req, res) => setImmediate(this.handler.bind(this, req, res, undefined), req, res)
+    const boundHandler = this.handler.bind(this)
+    this.attach = (req, res, next?: NextFunction) => setImmediate(boundHandler, req, res, next)
```

Comment:
no need to move it to a separate variable cuz it's used only once

Reply:
The difference is that `.bind` is invoked on every invocation when the variable is inlined - in theory, what I have done saves memory